### PR TITLE
Try @iblis17's suggestion for the travis time out

### PR DESCRIPTION
**Pull request by cstjean**
I hopefully got the syntax right. See also https://github.com/travis-ci/travis-ci/issues/7431

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,9 +15,11 @@ matrix:
 notifications:
   email: false
 # uncomment the following lines to override the default test script
-#script:
-#  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-#  - julia -e 'Pkg.clone(pwd()); Pkg.build("TraceCalls"); Pkg.test("TraceCalls"; coverage=true)'
+script:
+  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
+  - travis_wait() {
+         julia -e 'Pkg.clone(pwd()); Pkg.build("TraceCalls"); Pkg.test("TraceCalls"; coverage=true)'
```
Comment (iblis17) [inline .travis.yml]: oh, i meant `travis_wait julia -e 'Pkg.clone(pwd()); Pkg.build("TraceCalls"); Pkg.test("TraceCalls"; coverage=true)'`

the command `travis_wait` is provided by travis's environment.

**Comment by cstjean** [comment]
It seems to have [exited](https://travis-ci.org/cstjean/TraceCalls.jl/jobs/272633605#L498) immediately with success without running any of the tests.

**Comment by iblis17** [comment]
hmm ... no idea why

the sample of default testing commands: https://travis-ci.org/cstjean/TraceCalls.jl/jobs/272625174#L483

maybe this?
```yaml
- julia --color=yes -e 'Pkg.clone(pwd())'
- julia --color=yes -e "Pkg.build(\"${JL_PKG}\")"
- travis_wait 40 julia --check-bounds=yes --color=yes -e "Pkg.test(\"${JL_PKG}\", coverage=true)"
```

**Comment by cstjean** [comment]
Doesn't seem to work either... 

I fixed the issue on master by adding a print statement to my user-guide notebook. It's an ugly compromise, but it works. I'll ask tkelman about travis-wait next time I tag. Thank you for your help!

I'm sure there's a way in Julia to launch a background thread that prints to STDERR every 5 minutes for 40 minutes. That might be much easier than fighting travis.

**Pull request closed without merging**